proxy/http/Http1ServerSession.cc

```diff
@@ -204,36 +181,35 @@ Http1ServerSession::release()
   }
 }
 
-NetVConnection *
-Http1ServerSession::get_netvc() const
-{
-  return server_vc;
-};
-
-void
-Http1ServerSession::set_netvc(NetVConnection *new_vc)
-{
-  server_vc = new_vc;
-}
-
 // Keys for matching hostnames
 IpEndpoint const &
 Http1ServerSession::get_server_ip() const
 {
-  ink_release_assert(server_vc != nullptr);
-  return server_vc->get_remote_endpoint();
+  ink_release_assert(_vc != nullptr);
+  return _vc->get_remote_endpoint();
 }
 
 int
-Http1ServerSession::populate_protocol(std::string_view *result, int size) const
+Http1ServerSession::get_transact_count() const
 {
-  auto vc = this->get_netvc();
-  return vc ? vc->populate_protocol(result, size) : 0;
+  return transact_count;
 }
 
 const char *
-Http1ServerSession::protocol_contains(std::string_view tag_prefix) const
+Http1ServerSession::get_protocol_string() const
+{
+  return "http";
+}
+void
+Http1ServerSession::increment_current_active_connections_stat()
+{
+}
+void
+Http1ServerSession::decrement_current_active_connections_stat()
+{
+}
```

Comment:
Should these have the todo you have in other similar functions:

`  // TODO Implement stats